frontend/src/screens/settings/Settings.tsx

```diff
@@ -28,7 +28,7 @@ function Settings() {
   const { data: info } = useInfo();
 
   const [selectedCurrency, setSelectedCurrency] = useState<string | undefined>(
-    info?.currency
+    info?.currency.toUpperCase()
```

Comment:
can you fix the root cause? isn't it because the default currency set in the backend is `usd` instead of `USD`?

Reply:
resolved

Reply:
Is this change still needed then?

Reply:
removed